libs/cardano-ledger-test/src/Test/Cardano/Ledger/Generic/Consensus.hs

```diff
@@ -47,7 +48,7 @@ import Cardano.Ledger.Mary.Translation ()
 import Cardano.Ledger.Mary.Value (AssetName (..), MaryValue (..), MultiAsset (..), PolicyID (..))
 import Cardano.Ledger.PoolDistr
 import Cardano.Ledger.SafeHash
-import Cardano.Ledger.Shelley.API hiding (KeyPair, RequireAllOf, RequireAnyOf, RequireMOf, RequireSignature, vKey)
+import Cardano.Ledger.Shelley.API hiding (KeyPair, RequireAllOf, RequireAnyOf, RequireMOf, RequireSignature, Withdrawals, vKey)
```

Comment:
You see how much redundant work has to be done when names are not chosen properly. This should be like a golden rule: same codebase should never have clashing names.
```suggestion
import Cardano.Ledger.Shelley.API hiding (KeyPair, RequireAllOf, RequireAnyOf, RequireMOf, RequireSignature, vKey)
```

Reply:
:bow: